### ISSUE TITLE
Instance status page

### ISSUE THREAD

**Issue by Twixes** [PostHog/posthog#1812]
## Is your feature request related to a problem?

We currently have metrics.posthog.com, which offers crucial performance information, but is a solution separate from PostHog itself. Otherwise we don't offer any metrics, which makes detecting and explaining PostHog instance problems more time-consuming for users and also for the core team.

## Describe the solution you'd like

PostHog could use an instance status-type page (accessible only to organization admins on Self-Hosted or `is_staff` users on Cloud [or `is_superuser`? this distinction is actually useless for us]). For example, we could easily show database statistics like numbers of rows or table sizes, Redis health, event ingestion load – essentially data that comes up useful when users ask about problems.

## Describe alternatives you've considered

Tell users to investigate instance internals themselves.

## Additional context

Thought of this when a user needed to know why their PostHog Docker container was taking up so much space. I've told them to query for table sizes:
```SQL
SELECT relname as "table", pg_size_pretty(pg_total_relation_size(relid)) AS "size" FROM pg_catalog.pg_statio_user_tables ORDER BY pg_total_relation_size(relid) DESC;
```
but it'd be much simpler if this information was available in the UI.

**Comment by timgl** [comment]
The list of things we should include
- Event table size (both in MB and [Approximate size](https://wiki.postgresql.org/wiki/Count_estimate))
- Element table size (both in MB and [Approximate size](https://wiki.postgresql.org/wiki/Count_estimate))
- Redis queue size
- Possibly memory size of workers?
- Number of events in last day/week

**Comment by Twixes** [comment]
Closed by #1875.
